how-to-docs/dms-migration.md

```diff
@@ -0,0 +1,75 @@
+
+# **DMS Migration**
+
+### **Prerequisite**
+
+The following need to be true before you follow this guide:
+* AWS console logon
+* Access the DMS service from console
+* A region where STS has been activated
```

Comment:
`Access the DMS service from console` should this be Access TO the DMS service from THE console